src/SiteExtensions/LoggingBranch/LB.csproj

```
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <title>ASP.NET Core Extensions</title>
     <Description>This extension enables additional functionality for ASP.NET Core on Azure WebSites, such as enabling Azure logging.</Description>
-    <TargetFramework>$(DefaultNetFxTargetFramework)</TargetFramework>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework);netstandard2.0</TargetFrameworks>
```

More site extensions
```suggestion
    <TargetFramework>$(DefaultNetFxTargetFramework)</TargetFramework>
```